aap_chatbot/src/useChatbot/useChatbot.ts

```diff
@@ -139,6 +139,9 @@ const createGitHubIssueURL = (
   return url.toString();
 };
 
+// For fixing tooltips that pops up from an iframe
+export let bodyElement = document.body;
```

Comment:
I wonder whether it's better to return `bodyElement` from the hook vs adding this global? We've already had problems with global (and user state/messages). Are you certain this works ok? Will it still have a value when the `AnsibleChatbot` unmount?